### PR TITLE
CUD api for user Service

### PR DESCRIPTION
**Pull request by bshresthap**
CUD api for user Service

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,12 +27,59 @@ An `nvmrc` file is included if using [nvm](https://github.com/creationix/nvm).
...
 
 From within the root directory:
 
 ```sh
-npm install -g webpack
 npm install
+npm run seed
+npm run client-dev
+npm run start
 ```
```
Comment (jrozal) [inline README.md:85]: Updates to readme look good 👍. Consider adding responses.

File: server/database/helpers.js [hunk 2]
```diff
@@ -43,9 +43,65 @@ const getUserSuperhostStatus = async (userId) => {
...
 module.exports = {
   generatePhoto,
   getUserById,
   getUserNameAndPhoto,
-  getUserSuperhostStatus
+  getUserSuperhostStatus,
+  insertUserInfo,
+  updateUserInfo,
+  deleteUserInfo
 };
```
Comment (jrozal) [inline server/database/helpers.js:107]: new db functions look good

File: server/app.js [hunk 3]
```diff
@@ -31,8 +33,51 @@ app.get('/users/:userId/super', async (req, res) => {
...
+      if (results) {
+        res.status(200).send(results);
+      } else {
+        res.status(404).send({ message: 'Record was not deleted' })
+      }
+    })
+    .catch(err => console.log(err));
+
+})
+
```
Comment (jrozal) [inline server/app.js:82]: New routes look good

**Review by jrozal — APPROVED** [review]
Hi Barsha, new CUD operations look good 👍

File: README.md (hunk 1 above)
Comment (bshresthap) [reply README.md:85]: Good point. I will add it for next PR.

**Pull request merged**